### PR TITLE
fix when no from option is given

### PR DESCRIPTION
**Pull request by bendavies**
`hasOption` => `Returns true if an InputOption object exists by name.`

### CODE REVIEW

File: src/Command/AssertBackwardsCompatible.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ public function execute(InputInterface $input, OutputInterface $output) : int
         // @todo fix flaky assumption about the path of the source repo...
         $sourceRepo = CheckedOutRepository::fromPath(getcwd());
 
-        $fromRevision = $input->hasOption('from')
+        $fromRevision = $input->getOption('from') !== null
```
Comment (Ocramius) [inline src/Command/AssertBackwardsCompatible.php]: This kind of change requires a test addition

Comment (bendavies) [reply src/Command/AssertBackwardsCompatible.php]: Yep sorry, didn't see the command was tested. Will add.

Comment (bendavies) [reply src/Command/AssertBackwardsCompatible.php]: test already exists, updated it.

**Comment by Ocramius** [comment]
@bendavies I think #73 fixed this already (by using the correct API in `symfony/console`, which has broken `hasOption()`): closing as `duplicate` here, sorry :-\

**Pull request closed without merging**